scripts/check_frontend_test_coverage.py

```diff
@@ -0,0 +1,430 @@
+        """Sets the file_name property.
+        
+        Raises:
+            Exception: If total lines number is not found.
+        """
+        match = re.search('LF:(\d+)', total_lines_line)
+        if match == None:
+            raise Exception(
+                'It wasn\'t possible to get the total lines of {} file.'
+                'It\'s not possible to diff the test coverage correctly.'
+                .format(self.get_file_name()))
+        self.total_lines = int(match.group(1))
+
+
+    def set_covered_lines(self, covered_lines_line):
+        """Sets the file_name property.
+        
+        Raises:
+            Exception: If covered lines number is not found.
+        """
+        match = re.search('LH:(\d+)', covered_lines_line)
+        if match == None:
+            raise Exception(
+                'It wasn\'t possible to get the covered lines of {} file.'
+                'It\'s not possible to diff the test coverage correctly.'
+                .format(self.get_file_name()))
+        self.covered_lines = int(match.group(1))
+
+
+def run_frontend_tests_script():
+    """Run the frontend tests script using subprocess. If any test fails then
+    the coverage check won't happen.
+    """
+
+    try:
+        subprocess.check_call([
+            'python', '-m', 'scripts.run_frontend_tests'])
+    except subprocess.CalledProcessError:
+        sys.exit(1)
+
+
+def filter_relevant_lines(line):
+    """Check if the line has file path (SF) or total lines (LF) or covered
+    lines (LH) of the test.
+
+    Args:
+        line: str. A line from lcov file.
+
+    Returns:
+        Boolean. If the line has the file path or total lines or covered lines
+        of the test.
+    """
+    return (
+        line.startswith('SF') or line.startswith('LH')
+        or line.startswith('LF'))
+
+
+def get_stanzas_from_lcov_file():
+    """Get all stanzas from a lcov file. The lcov file gather all the frontend
+    files that has tests and each one has the following structure:
+    TN: test name
+    SF: file path
+    FNF: total functions
+    FNH: functions covered
+    LF: total lines
+    LH: lines covered
+    BRF: total branches
+    BRH: branches covered
+    end_of_record
+
+    Returns:
+        list(Stanza). A list with all stanzas.
+    """
+
+    f = python_utils.open_file(LCOV_FILE_PATH, 'r')
+    lcov_items_list = f.read().split('end_of_record')
+    stanzas_list = []
+
+    for item in lcov_items_list:
+        relevant_lines = [line for line in item.splitlines() if 
+            filter_relevant_lines(line)]
+
+        if len(relevant_lines) > 0:
+            stanza = LcovStanzaRelevantLines()
+
+            stanza.set_file_name(relevant_lines[0])
+            stanza.set_total_lines(relevant_lines[1])
+            stanza.set_covered_lines(relevant_lines[2])
+
+            stanzas_list.append(stanza)
+
+    return stanzas_list      
+
+
+def remove_item_from_whitelist(whitelist, test_name):
```

Comment:
You don't need a separate function for this. Just inline it into the main function.

Also I think you can use whitelist.remove()? I believe it does the same thing.

Reply:
Done.